dataactcore/models/jobModels.py

```diff
@@ -35,8 +49,10 @@ class Submission(Base):
     datetime_utc = Column(DateTime)
     user_id = Column(Integer, nullable=False) # This refers to the users table in the User DB
     cgac_code = Column(Text)
-    reporting_start_date = Column(Date)
-    reporting_end_date = Column(Date)
+    reporting_start_date = Column(Date, nullable=False)
+    reporting_end_date = Column(Date, nullable=False)
+    reporting_fiscal_year = Column(Integer, nullable=False, default=generateFiscalYear, server_default='0')
```

Comment:
Admittedly I'm not fully conversant in sqlalchemy `default` vs `server_default` but is there a reason both wouldn't be set to your `generateFiscalYear` function?